docs/rest.md

```diff
@@ -670,7 +670,11 @@ Sample input:
 ```javascript
 {
     “action”: “connect”,
-    “plug”: {“snap”: “canonical-pi2”,   “plug”: “pin-13”},
-    “slot”: {“snap”: “keyboard-lights”, “slot”: “capslock-led”}
+    “plugs”: {
+        {“snap”: “canonical-pi2”,   “plug”: “pin-13”},
+    },
+    “slots”: {
+        {“snap”: “keyboard-lights”, “slot”: “capslock-led”},
+    }
```

Comment:
also, I didn't notice before, but typographic quotes have snuck in to the json examples. Please use straight quotes in code.